### DISCUSSION TITLE
Как получить YOLO значок?

### DISCUSSION THREAD

**Discussion by Dl82d** [community/community#138698] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Объясните, пожалуйста

**Comment by Shipy4kaRU** [comment, marked as answer]
Он выдается тебе, если хотя бы один твой pull request был принят без замечаний (автор не написал ни одного треда и смёржил правки). Т.е чел стоит в Collaborator, ты добавляешь пулл реквест, а затем мержишь его, при этом Collaborator не должен написать тебе ничего

**Comment by LiteBrite82** [comment]
Hi @Dl82d , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
